docs/interactions/Receiving_and_Responding.md

```diff
@@ -248,15 +256,32 @@ An interaction is the base "thing" that is sent when a user interacts with your
-| custom_id?      | string                                                                                                                                                                                 | the [`custom_id`](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/custom-id) of the component                                             | Component          |
-| component_type? | integer                                                                                                                                                                                | the [type](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-types) of the component                                              | Component          |
-| values?         | array of [select option values](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-option-structure)                                                                      | the values the user selected                                                                                                    | Component (Select) |
+| Field           | Type                                                                                                                                                                                         | Description                                                                                                                           | Interaction Type |
+| --------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
+| id              | snowflake                                                                                                                                                                                    | the [`ID`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command   | Slash Command    |
+| name            | string                                                                                                                                                                                       | the [`name`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command | Slash Command    |
+| resolved?       | [resolved data](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-resolved-data-structure)                                                                                      | converted users + roles + channels                                                                                                    | Slash Command    |
```

Comment:
These apply to all Application Commands right?

Reply:
Yes, thanks!